chore: Add article about retrieving the list of default images for Im… (#2795)
codenvy/che-docs: modules/administration-guide/pages/retrieving-default-list-of-images-for-kubernetes-image-puller.adoc

:_content-type: PROCEDURE
:description: Retrieving the default list of images for {image-puller-name}
:keywords: administration-guide, image-puller
:navtitle: Retrieving the default list of images for {image-puller-name}

[id="retrieving-default-list-of-images-for-kubernetes-image-puller"]
= Retrieving the default list of images for {image-puller-name}

> This article provides instructions on retrieving the default list of images used by {image-puller-name}. This can be helpful for administrators who want to review and configure {image-puller-name-short} to use only a subset of these images in advance.

Reviewer: ```suggestion
Learn how to retrieve the default list of images used by {image-puller-name}. This can be helpful for administrators who want to review and configure {image-puller-name-short} to use only a subset of these images in advance.
```
